src/sage/quadratic_forms/quadratic_form__mass__Conway_Sloane_masses.py

```diff
@@ -115,7 +115,7 @@ def is_even(self, allow_rescaling_flag=True):
     Return true iff after rescaling by some appropriate factor, the
     form represents no odd integers.  For more details, see :meth:`parity`.
 
-    Requires that `Q` is defined over ``ZZ``.
+    Requires that Q is defined over ``ZZ``.
```

Comment:
You are systematically removing single-quotes from `Q`. Why? `Q` represents a quadratic form, and hence is a math.

Reply:
My reading of the existing markup is that Q is intended to be set upright, not italic. And this is how the existing markup is doing that.

Reply:
I'd be happy to make all occurrences of Q systematically math italic if that's what you prefer.